modules/scanning-images.adoc

```diff
@@ -70,68 +69,85 @@ Scanner can check for vulnerabilities in images that use the following package f
 * dpkg
 * RPM
 
-[discrete]
+[id="supported-programming-languages_{context}"]
 == Supported programming languages
 
 Scanner can check for vulnerabilities in dependencies for the following programming languages:
 
+* Go (supported in Scanner V4 only)
 * Java
 * JavaScript
+* NodeJS (supported in Scanner V4 only)
 * Python
 * Ruby
 
-[discrete]
+[NOTE]
+====
+The vulnerability source for languages, link:https://osv.dev/list[Open Source Vulnerabilities (OSV)], provides GitHub Advisory (GHSA) IDs rather than CVE numbers for vulnerabilities. OSV is used for some Scanner V4 results. The {product-title-short} Scanner V4 uses the OSV database available at OSV.dev under license.
```

Comment:
This can probably be moved to where we list all vuln sources (commented way above)